src/serializer/functions.js

```diff
@@ -295,19 +350,28 @@ export class Functions {
     return call.bind(this, globalThis, args);
```

Comment:
Does this mean that the actual binding is effectively ignored? Surprising.

Reply:
It should be on bounding functions, you can't apply a new context to them as per spec.